runtime/onert/core/src/compiler/Fp32ToFp16Converter.cc

```diff
@@ -290,7 +302,19 @@ void Fp32ToFp16Converter::appendNewOpSeqForConvertFp16ToFp32(const ir::OpSequenc
 {
   // OpSeq's output set is included in the last operation's output set
   const ir::OperandIndexSequence op_seq_outputs = op_seq.getOutputs(); // copied
-  for (const auto &op_seq_output_ind : op_seq_outputs)
+
+  // Remove duplicated input
+  // NOTE Please do not change sequence of op_seq_outputs. It can change the sequence of outputs of
```

Comment:
Hmm.. if the sequence is important, I think `list` structure is better.

Reply:
This is `OperandIndex`. It is not `OperationIndex`.